tests/test_uploads.py

```diff
@@ -80,8 +80,8 @@ def testGet(self):
         self.assertEqual(rv.status_code, 404)
 
     def testRegularUploads(self):
+        '''Test that uploaded file return the expected status code.'''
```

Comment:
Looks better! One small grammar nit: `file return` should probably be `files return`.